src/main/java/uk/gov/hmcts/sptribs/systemupdate/service/CcdSearchService.java

```diff
@@ -37,8 +37,6 @@ public class CcdSearchService {
     @Autowired
     private CoreCaseDataApi coreCaseDataApi;
 
-
-
     public List<CaseDetails> searchForAllCasesWithQuery(
         final BoolQueryBuilder query, User user, String serviceAuth, final State... states) {
```

Comment:
There is a dummy/test case reference 1695290772211061L added in the java code under the method searchForCasesWithVersionLessThan. This has to be removed and the corresponding test needs to be adjusted as required.